libtransmission/peer-mgr.cc

```diff
@@ -1899,6 +1901,7 @@ void tr_peerMgr::reconnectPulse()
     auto const now_sec = tr_time();
 
     // remove crappy peers
+    auto bad_peers_buf = bad_peers_t{};
```

Comment:
Perhaps this should be a static variable in `close_bad_peers()` if we are going to construct and destruct twice a second. 🤔

Reply:
I'm not saying "never", but I'd prefer to avoid static fields like this unless there's a *big* win, and I don't see a big win here since this `small::vector` will keep most of the action on the stack here.